packages/terra-notification-dialog/tests/wdio/notification-dialog-spec.js

```diff
@@ -122,7 +122,7 @@ Terra.describeViewports('NotificationDialog with additional focus trap sources w
 
     // it opens notification dialog
     browser.click('#trigger-notification-dialog');
-    Terra.validates.element('Open notification dialog', { selector: '[class*="abstract-modal"]' });
+    Terra.validates.element('Open notification dialog', { selector: '[aria-label="Notification Dialog"]' });
```

Comment:
This won't actually update the selector for accessibility. the accessibility selector will always be document.

see: https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraCommands/axe-command.js#L37